test/associative/ferguson/assoc-sort-comparator.chpl

```diff
@@ -19,12 +19,12 @@ A[5] = 5;
 A[7] = 3;
 
 writeln("reverse sorted D");
-for i in sorted(D, reverseComparator) {
+for i in sorted(D, new ReverseComparator()) {
   writeln(i);
 }
 
 writeln("reverse sorted A");
-for x in sorted(A, reverseComparator) {
+for x in sorted(A, new ReverseComparator()) {
```

Comment:
should these be `reverseDefaultComparator`? And same with all the other places its used

Reply:
Not that it really matters that much from a functional point of view (since `ReverseComparator` defaults to `DefaultComparator`)

Reply:
You actually can't use the type alias with `new` due to the generic field being a `var`